.semaphore/prologue.sh

```diff
@@ -15,21 +22,11 @@
 # some places (which we can't change).
 set -evo pipefail
 
-# Clear disk space by removing docker related services, mounts and files.
-sudo systemctl stop docker.service docker.socket
+# Install Nix. We install in single-user mode (--no-daemon) because the Nix
+# process can access the running SSH agent to fetch private Git repositories.
+curl -o install-nix https://releases.nixos.org/nix/nix-2.8.1/install
```

Comment:
Is there something specific about Nix 2.8.1 that we rely on or can we bump this higher? IIRC there's something finicky between 2.10-11 but I can't seem to remember what exactly it was.

Reply:
To be honest I have no idea. I just saw that other projects used this version so I went with it, but after further investigation I also see projects using `2.11.1` so I'll just bump it up to that. Thanks for noticing!